sag_et.par

```diff
@@ -4,195 +4,241 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/||\
 n 7 4
+/||\
+n 9 5
+|\
 n 10 6
-/|)  /N\
+(|\  /||\
 d 12 7
 s 2 3 4 6
-/N\
+/||\
 n 14 8
 |\
 n 15 9
-/|  N\  /N\
+/|  ||\  /||\
 n 16 9
-)N(  M)
+|)  (|\
 d 17 10
-/|\  /N\
+/|\  /||\
 d 19 11
-/N\
+/||\
+n 20 12
+)~|  (|\  ~||\  /||\
 n 21 12
-|)  |\
+|)  (|\
 d 22 13
 s 11
-/|  N\  /N\
+/|  ||\  /||\
 n 23 13
-)N(  M)  (X(
+|)  |\  (|\
 d 24 14
 s 8
-/|\  /N\
+/|\  /||\
+n 25 15
+)~|  /|  ||\  ~||\  /||\
 d 26 15
-/N\
+s 13
+/||\
 d 27 16
-/|  /|)  N\  /N\
+/|  (|\  ||\  /||\
+n 28 16
+|)  |\  (|\
 d 29 17
-/|  N\  /N\
+/|  ||\  /||\
+n 30 18
+)|  /|  (|\  ||\  (||~  /||\
 d 31 18
-/|\  /N\
+/|\  /||\
+n 32 19
+)~|  /|  ||\  ~||\  /||\
+n 33 19

```

Comment:
In the periodic table chart on the front page of sagittal.org, the bit that says "Don't use flat or sharp here" only extends to cover to the point of the first five 7n-EDOs. However, as best as I could tell from my work [here](https://docs.google.com/spreadsheets/d/1Xnm2yNQ2C8QuD7Di-Ik7ttmni-AjFmAkDN2clQbzuuo/edit#gid=210239964) the other three EDOs which are in the shown nearby in the same salmon color should also not use the apotome. I'm concerned that maybe some of the boundaries of the information in the final post on the forum thread [here](http://forum.sagittal.org/viewtopic.php?p=845#p845) were ambiguous and didn't quite translate correctly into that chart. To be specific, the three EDOs in question are 33, 40, and 47. I did not include the apotome in their symbol set even though the periodic table seems to say to do so.

In the case of 52EDO, where the limma is much bigger than the apotome, we barely squeak by, using the reach of the double-apotome to connect nominals separated by whole tones with only a single sagittal.

But in the case of any of these three, with only a single step to the apotome, the sequence of sagittals emanating from the base nominal would surpass the apotome, and the sequence of sagittals emanating from the apotome would overlap with that sequence, an effect which occurs nowhere else. Either that, or the sagittal symbol sequence was meant to only exist as applied on top of the apotome symbol, but that was also unprecedented. The simplest interpretation I could come up with was that the apotome was not meant to be used for these three EDOs either, even though they have a >0 apotome size.

Reply:
You're quite right that this is an error in the front page periodic table. #, b, /||\, \!!/ should not be used for _any_ of the rose/salmon coloured EDOs. It's quite likely that Scala will need to be modified to handle them correctly. Scala will be able to tell they are limma-fraction notations because the symbol set will only go up to /N (the limma symbol), not /N\ (the apotome). The periodic table only lists up to the largest single-shaft symbol. The remaining symbols must be looked up from a list of limma-complements as shown near the end of this post. (Note that this does not have the final notations) http://forum.sagittal.org/viewtopic.php?p=781#p781

Reply:
I took a note to fix the bug in the periodic table. (You did mean #, b, /|||, \!!/, though, right?)

I have also taken a note to alert Manuel to these limma-fraction notations. So I take it that my changes to the .par file are correct? The limma-complements would be handled, most likely, wherever else in the Scala code where apotome-complements are already being handled.

I admit I'm quite curious how to use the limma-complements to extend these rose-colored EDOs into double-shaft territory, even though it doesn't seem to ever be necessary, as single shafts emanating from neighboring nominals always look like they surpass each other. I'd love if you could say more about it.

Reply:
I think "markdown" is eating our backslashes. And you've made a typo. I meant #, b, /||\ and \!!/.
See http://forum.sagittal.org/viewtopic.php?p=1747#p1747 for the full Revo limma-fraction notations.

Reply:
Aw man, so it is! And so I did!

Thanks for that forum post. That really clarifies things. It's interesting how the Revo notation is limited (limmated?!) here where Evo has the advantage. Although I suppose we can't blame it on any shortcomings of Sagittal to make things fair between Evo and Revo; if anything, it's the lack of a historical/conventional symbol for a limma! I suppose if anyone were to know something about that, it'd be you.

I'll make those changes to the .par files in the next commit. I've also taken a note to keep this in mind when iterating on the Periodic Table.

By the way: as we know well, single-shafts cover about 60% of the apotome, and it appears they cover about 76% of the limma.

Reply:
Didn't you, like Joe Monzo recently, just make a case against the use of the terms Evo and Revo, by getting them swapped? Didn't you mean to say that Revo has the advantage in the case of limma-fraction notation, because Evo has a more "limmated" range. No, I've never heard of a limma symbol outside of Sagittal. Not much call for it as, like an 8:9 major whole-tone, it can be notated by a change of nominal (possibly with sharps or flats).

Not surprising that the single-shafts cover more of a limma, since a limma is smaller than an apotome ... by about 60/76 = 0.789 (90.225¢/113.685¢ = 0.794).